tool/teleport/common/teleport.go

```diff
@@ -810,6 +811,8 @@ Examples:
 		err = onGetLogLevel(ccf.ConfigFile)
 	case collectProfilesCmd.FullCommand():
 		err = onCollectProfiles(ccf.ConfigFile, ccf.Profiles, ccf.ProfileSeconds)
+	case readyzCmd.FullCommand():
+		err = onReadyz(ccf.ConfigFile)
```

Comment:
```suggestion
		err = onReadyz(ctx, ccf.ConfigFile)
```

We should propagate the parent context.

Reply:
Done